### ISSUE TITLE
Trucs et astuces pour accélérer la transcription

### ISSUE THREAD

**Issue by Bentham** [jlm2017/jlm-video-subtitles#129]
Bonjour à tous,

Je vous propose cette petite vidéo, dans laquelle je vous montre mes petites astuces pour transcrire un peu plus vite. N'hésitez pas à donner les vôtres dans les commentaires.

J'espère que ça vous sera utile !

## Vidéo
https://youtu.be/tg5NzbiRqI0

## Chapitres de la vidéo
- [Présentation de la vidéo](https://youtu.be/tg5NzbiRqI0?t=0s)
- [Rappel des raccourcis clavier](https://youtu.be/tg5NzbiRqI0?t=16s)
- [Comment faire quand on commence la rédaction d'un sous-titre ?](https://youtu.be/tg5NzbiRqI0?t=58s)
- [Comment faire quand on corrige l'orthographe des sous-titres ?](https://youtu.be/tg5NzbiRqI0?t=520s)
- [Comment faire quand on repasse pour un contrôle global ?](https://youtu.be/tg5NzbiRqI0?t=604s)
- [Conclusion de la vidéo](https://youtu.be/tg5NzbiRqI0?t=640s)

## Méthode alternative, plus efficace (avec reconnaissance vocale)
https://youtu.be/q68KAKVCOP4

### Racourcis usuels d'édition de texte
Raccourci | Fonction
-----------| -----------
`<Ctrl> + <X>` | Couper le texte dans le presse papier
`<Ctrl> + <C>` | Copier le texte dans le presse papier
`<Ctrl> + <V>` | Coller le texte
`<Ctrl> + <A>` | Tout sélectionner
`<Début>` | Positionner le curseur au début du texte
`<Fin>` | Positionner le curseur à la fin du texte
`<Ctrl> + <Flèche de droite>` | Positionner le curseur à la fin du mot
`<Ctrl> + <Flèche de gauche>` | Positionner le curseur au début du mot
`<Ctrl> + <Maj> + <Flèche de gauche>` | Sélectionner le mot avant le curseur 

### Racourcis propres au sous-titrage YouTube
Raccourci | Fonction
-----------| -----------
`<Maj> + <Flèche de gauche>` | Revenir en arrière
`<Maj> + <Flèche de droite>` | Avancer d'une seconde
`<Maj> + <Espace>` | Mettre en pause la vidéo ou lancer la lecture
`<Maj> + <Entrée>` | Insérer un saut de ligne dans le sous-titre
`<Maj> + <Flèche du bas>` | Passer au container de sous-titres suivant
`<Maj> + <Flèche du haut>` | Passer au container de sous-titres précédent
`<Entrée>` | Ajouter un nouveau container de sous-titres

**Comment by Flupignon** [comment]
Super merci @Bentham !!!

**Comment by Albapringe** [comment]
Je t'ai laissé un commentaire sur ta vidéo YT pour te répondre ;) J'ajoute ici que je confirme que cette vidéo devrait être un détour nécessaire pour tous ceux qui débutent dans le sous-titrage.

**Comment by Flupignon** [comment]
Comment tu gères @Bentham  merci ! Très instructif !

**Comment by Bentham** [comment]
Bon c'est chouette si ça vous sert :)

**Comment by LVAI** [comment]
Merci @Bentham. Super vidéo ! Ça aidera à coup sûr tous les débutants en sous-titrage. 
J'ajouterai avec ma maigre expérience le ctrl+A pour sélectionner tout le texte, plutôt que le fin / ctrl+début, non ?
Et que l'on peut taper et valider (entrer) un long paragraphe si on préfère puis faire entrer dans la case de correction pour découper le sous-titres en plusieurs.
Enfin j'ai été étonnée de te voir récrire du texte que j'avais déjà tapé. Vous savez comment YouTube gère le travail à plusieurs sur des mêmes sous-titres ? Mal j'imagine sinon vous ne passeriez pas par GitHub ^^ mais combien de temps il met pour mettre en visibilité les sous-titres écrits et ce qu'il se passe si plusieurs personnes écrivent les mêmes sous-titres ?

**Comment by Bentham** [comment]
Oui tu as raison le contrôle+A est sans doute plus simple !

La gestion du travail à plusieurs est assez obscure et chaotique... Ce qui occasionne parfois des pertes considérables de travail ! Il faut bien penser à envoyer son texte dès qu'on arrête de bosser - même si on n'a pas terminé - et le signifier au groupe en changeant le label sur github. Il faut savoir que l'enregistrement automatique n'enregistre le brouillon que pour le rédacteur lui-même, pas pour les autres.

**Comment by DevilFishGold** [comment]
Je ne t'ai pas remercié plus tôt car je voulais finir le sous-titrage d'une vidéo mais merci ^^ 
Sans ton aide ça aurait pris plusieurs jours à faire. Je n'avais même pas vu les raccourcis =_="
Excellente vidéo, pédagogique en plus. Bravo! :)

**Comment by LVAI** [comment]
Je note le bouton "envoyer pour examen" que je n'avais pas repéré (!) et qui permet de valider son travail qui n'est sinon sauvegardé qu'en local.
Et j'ajoute un autre raccourcis, au cas où, pour la gestion de texte au clavier : le ctrl+flèches (gauche ou droite) qui permet de se déplacer dans son texte mot par mot.
Sinon, je pense que ça pourrait être pas mal de faire une charte pour la rédaction des sous-titres afin d'uniformiser les pratiques (lol la fille qui débarque et qui réforme tout...) : les points de suspension ou pas quand on coupe une phrase, couper ou pas les phrases, les informations sur les bruits à mettre ou non, la simplification du texte pour une meilleure lecture, etc.

**Comment by DevilFishGold** [comment]
Le problème c'est qu'il y a des personnes qui ont des pratiques différentes. Je me prend comme exemple pour illustrer: je pense qu'il y a des personnes qui ne font pas du tout comme moi, c'est à dire de retranscrire les interjections etc. Je le fais car j'estime que le discours de JLM se base sur ça. Il n'établit pas son discours de la même manière qu'un Fillon par exemple. Jean-Luc utilise abondamment les interjections, les "eh bien", etc. Et je trouverai ça dommage que les personnes étant dans l'incapacité de pouvoir entendre cela, qui je pense est quelque chose d'important dans le discours de JLM, ne puisse pas les lire aussi.
Sans parler du fait que des gens ne participant avec nous sur cette plateforme puissent sous-titrer aussi. Laisser les gens faire comme ils veulent serait bien plus appréciable je pense. :)

**Comment by Flupignon** [comment]
Oui son oralité est très importante @DevilFishGold 
Pour les raccourcis il y a aussi ctrl+shift+flèche (gauche ou droite) pour sélectionner mot par mot. Avec ctrl+flèche on gagne v'la le temps.

**Comment by Flupignon** [comment]
Hey l'hologramme ça fait le buzz dis donc ! Tout les autres en PLS x)

**Comment by Bentham** [comment]
Can't stenchon the holochon !

**Comment by LucileDT** [comment]
@Bentham Merci beaucoup pour cette vidéo, j'ai rajouté un petit commentaire dessus. Je vais rajouter une entrée à la FAQ et un lien vers cette issue dans les modes d'emploi pour démarrer :D

**Comment by LVAI** [comment]
Forte de mes 8 minutes de sous-titrage d'expérience XD je voulais partager ma technique qui me fait un temps précieux pour le calage des sous-titres :
Je saisi un texte long (1 à 2 minutes de discours) puis je le valide pour créer le sous-titre associé. Ensuite j'étends ce sous-titre jusqu'au précédent, je me mets en édition - YouTube me place automatiquement au bon endroit de la vidéo - et je lance la lecture - maj + espace. A chaque fois qu'un morceau de phrase cohérent a été dit, je mets la vidéo en pause  - maj + espace - et je tape "Entrée". YouTube découpe automatiquement le sous-titre et l'arrête à l'endroit où j'ai fait pause. Il est donc bien calé ! Je relance la lecture et continue ainsi jusqu'à avoir découpé tout mon texte.

**Comment by Bentham** [comment]
Tiens c'est intéressant comme manière de faire, je vais essayer !

**Comment by Albapringe** [comment]
Oh purée, ça marche du tonnerre ce truc ! Alléluia, tu viens de me faire ma nuit et le reste de ma semaine !

**Comment by Bentham** [comment]
Testé et approuvé ! Je vais refaire la vidéo d'ici peu avec cette méthode, qui m'a l'air plus efficace. N'hésitez pas à me donner vos propres trucs et astuces, je les inclurai s'il permettent d'accélérer encore le processus !

**Comment by LucileDT** [comment]
@Bentham Tu éditeras cette issue pour y mettre le lien de la nouvelle vidéo ?

**Comment by LVAI** [comment]
Et pour info, un article avec des bonnes pratiques pour le sous-titrage : [web-pour-tous](http://www.web-pour-tous.org/spip.php?article140)
Je respecte l'idée que chacun soit libre de faire comme il veut bien sûr :)
Donc aucune obligation, juste de l'information pour ceux qui voudraient.
On ne peut pas tout mettre en place avec YouTube (les codes couleurs en particulier, qui sont pourtant très importantes, le positionnement, ...) mais ça permet d'avoir une idée de comment faire des sous-titres plus efficaces, en particuliers : la longueur des sous-titre, la durée, l'organisation des dialogues, espacement.
D'ailleurs je pense que le _blank_ dont ils parlent correspond à peu près à la bande bleue qui permet de positionner les sous-titres.

**Comment by Bentham** [comment]
J'ai mis à jour avec une nouvelle vidéo !

**Comment by LucileDT** [comment]
Magnifique tapis de souris ;)

**Comment by Bentham** [comment]
On est gamer ou on l'est pas !  :D

**Comment by LucileDT** [comment]
Est-ce que ça te dérangerait que j'édite ton issue ? Pour rajouter la liste des raccourcis claviers que tu présentes.

**Comment by Bentham** [comment]
Aucun pb. Et en plus je me plante sur la moitié des raccourcis que je donne, en disant "control" à la place de "shift"... :/

**Comment by LucileDT** [comment]
Je crois qu'il existe des logiciels qui permettent d'afficher à l'écran les touches sur lesquelles tu appuies quand tu appuies dessus, ça serait plus facile à voir que ton clavier filmé, tu veux que je me renseigne ?

**Comment by DevilFishGold** [comment]
@LVAI Concernant le code couleur, on peut mettre en place des couleurs sur YouTube?

**Comment by LucileDT** [comment]
@DevilFishGold Nope, ce n'est pas possible pour l'instant (je ne sais pas si on pourra le faire un jour).

**Comment by DevilFishGold** [comment]
@LucileDT Ok ok, merci :)

**Comment by LucileDT** [comment]
Raccourcis rajoutés !

**Comment by LVAI** [comment]
Trop cool la nouvelle version de la vidéo ! (et la dédicace :p)
@DevilFishGold  Non, come le dit LucileDT pour le moment les possibilités de YouTube sont limitées : pas de code couleur, pas de positionnement des sous-titres sur la vidéo. Mais on peut toujours rêver un peu.

**Comment by Vimogo** [comment]
Merci pour les conseils Bentham !

**Comment by Bentham** [comment]
Je viens de rajouter une deuxième vidéo, sur une méthode alternative avec reconnaissance vocale. Plus efficace, mais peut-être plus complexe à mettre en place (encore que...) : https://youtu.be/q68KAKVCOP4

**Comment by LVAI** [comment]
Encore une super vidéo @Bentham ! Tu fais vraiment du super boulot.
Je vais tester la reconnaissance vocale aujourd'hui !
Tu mets ta vidéo à quelle vitesse pour la saisie ? Tu as testé en vitesse normale, j'imagine, c'est pas jouable ?
PS : C'est L.V.A.I. (comme les 4 lettres ^^)

**Comment by Bentham** [comment]
Pour moi c'est la vitesse 0,5 qui me semble confortable pour la saisie, mais je t'invite à faire des essais de ton côté, tu me diras :)

**Comment by LVAI** [comment]
Oulala faut que j'arrête de mettre "super" dans toutes mes phrases :-D
Je vais dire sensas' comme Noelle, à la place !
Méthode de la diction testée et approuvée. Elle n'est pas facile à prendre en main au début je trouve. Il faut dire aussi que je n'ai qu'un PC et un casque, pas du matos de pro ^^
Mais au final on s'y fait. Elle est loin d'être infaillible aussi mais elle permet d'avoir un long texte pré-saisi, même si on saute un peu des mots, de la mise en forme, il ne reste plus qu'à faire quelques corrections directement dans l'éditeur Youtube.

**Comment by Bentham** [comment]
Sensas' !

**Comment by LucileDT** [comment]
N'y a-t-il pas moyen de faire de la reconnaissance vocale directement avec le son de la vidéo lorsqu'il est de bonne qualité, en utilisant un micro virtuel qui diffuserait ce qui est censé être la sortie son ?

**Comment by Bentham** [comment]
Je n'ai pas essayé, mais c'est très improbable. Pour t'en convaincre, je t'invite à activer les sous-titres automatiques de youtube, qui correspondent exactement à ce processus :)

**Comment by LucileDT** [comment]
@Bentham je ne sais pas si les deux utilisent le même moteur. Je ferai un test comparatif si tu veux pour vérifier.

**Comment by Bentham** [comment]
Ca pourrait être très intéressant oui, si tu as le temps !

**Comment by LucileDT** [comment]
Bon, testé, et bah c'est nul effectivement :( Je suis déception !

**Comment by Bentham** [comment]
Tu arrives à peu près au même résultat qu'avec la trad auto ?

**Comment by LucileDT** [comment]
Oui, c'est exactement ça. Je me demande pourquoi on obtient un rendu aussi crade, peut-être que YouTube encode le son d'une manière qui ne convient pas au robot ?

**Comment by Goutte** [comment]
Bonne idée, ce tutoriel !

J'ajouterais 

```
<Ctrl> + <Maj> + <Flèche de gauche> : Sélectionner le mot avant le curseur 
```

Je l'utilise tout le temps, celui-là  :sunglasses:

**Comment by LucileDT** [comment]
Rajouté !

**Comment by desintoxiquezvous** [comment]
bravo @Bentham pour ces infos , je pratique comme ça ,suite a tes infos sur la réalisation des S/T: 
la technique de LVAI avec la découpe automatique S/T par S/T est top!
une remarque : il ne faudrait pas  dans la mesure du possible, dépasser 2 lignes de sous titres à l’écran et c'est facile car on voit sur le film l'endroit ou on peut faire la césure, pourquoi ? et bien , lorsqu'on lit la vidéo sous titrée dans le logiciel Youtube sur IPAD les sous titres n'apparaissent (souvent) que SUR UNE SEULE LIGNE (en petits caractères) ! et du coup si les S/T sont trop longs... c'est coupé à la fin, et ce, jusqu’à l'apparition du sous titre suivant
je ne sais pas à quoi est dû la différence entre la version  classique sur ordi et la version ipad 
aucun réglage dans Youtube (sur Ipad) ne permet de modifier la taille des polices et donc de tout voir (en tout cas je n'ai pas trouvé).

autre suggestion: je pratique donc selon ta méthode la reconnaissance vocale dans Google docs ce qui me permet d’enregistrer le texte au km dans un fichier texte (excellente méthode) , j'ai tendance a faire une grande partie du film ainsi (pour "le goût de la mer" j'ai tout fait (environ 1h 30 de travail avec les plus grosses retouches en direct pour une vidéo de 16 mn 30 )
ne serait-il pas judicieux de mettre ce fichier texte (si quelqu'un a déjà fait le boulot) à disposition dans l'issue correspondante; du coup les suivants pourraient piocher dedans et gagner du temps
de saisie ?
voici par exemple le pdf du texte "le goût de la mer" qui m'a servi https://github.com/Bentham
[Texte le goût de la mer.pdf](https://github.com/jlm2017/jlm-video-subtitles/files/736905/Texte.le.gout.de.la.mer.pdf)

**Comment by Goutte** [comment]
@desintoxiquezvous Si on ferme cette issue, ne devrait-on pas en faire une page de wiki, ou une page du dépôt ?

(je veux bien le faire, mais j'ai pas les droits)

**Comment by LVAI** [comment]
Merci @desintoxiquezvous !
Les deux méthodes reconnaissance vocale + découpage des sous-titres après coup fonctionne pas mal.
Moi j'ai un peu de mal avec la reconnaissance vocale mais je trouve quand même que ça marche bien et ça va beaucoup plus vite.

Pour le découpage, j'approuve, comme d'habitude le 2 lignes max. Attention néanmoins à ne pas découper à deux lignes piles mais respecter le sens et le rythme de la phrase pour faciliter la lecture.
L'argument de la tablette est très intéressant. Il faut absolument le prendre en compte selon moi. @Bentham un avis ?

Je suis également d'accord pour mettre le texte à disposition.
Pour moi la difficulté est que je fais les retouches dans YouTube donc le texte Google est de moyenne qualité.

**Comment by LucileDT** [comment]
Il est possible que @desintoxiquezvous se soit trompé de bouton. Cette issue étant toujours source de discussion et d'échanges, je la rouvre dans le doute.

**Comment by LucileDT** [comment]
@Goutte tu es ajouté aux contributeurs, tu pourras éditer la documentation si tu le souhaites :) *(j'ai bien vu ta pull request, malheureusement je ne peux pas la valider, je t'explique dans ton commit pourquoi)*

@desintoxiquezvous je suis tout à fait d'accord pour la mise à disposition du texte.
